main/say.c

```diff
@@ -1293,6 +1300,158 @@ static int ast_say_number_full_de(struct ast_channel *chan, int num, const char
 	return res;
 }
 
+/*! \brief  ast_say_number_full_ps: Pashto syntax
+
+ New files:
+ In addition to English, the following sounds are required:
+ - wesht
+ - sawa
+ - zara
+ */
+static int ast_say_number_full_ps(struct ast_channel *chan, int num, const char *ints, const char *language, const char *options, int audiofd, int ctrlfd)
+{
+	int res = 0, t = 0;
+	int mf = 1;                            /* +1 = male and neuter; -1 = female */
+	char fn[256] = "";
+	char fna[256] = "";
+	if (!num)
+		return ast_say_digits_full(chan, 0, ints, language, audiofd, ctrlfd);
+
+	if (options && (!strncasecmp(options, "f", 1)))
+		mf = -1;
+
+    int playWesht = 0;
```

Comment:
Also messed up here. And you can't declare variables after use, compile with developer mode and your code will probably fail to build.

You need to fix all the whitespace and variable misuse in this change.

Reply:
oh, crap looks like my vim settings messed up white spaces / tabs.